### PR TITLE
feat(scalars): add a checkbox field

### PR DESCRIPTION
**Pull request by YasielCabrera**
- Created new checkbox fields
- Improved form label component
- Created tooltip component

### CODE REVIEW

File: packages/scalars/package.json [hunk 1]
```diff
@@ -49,15 +49,15 @@
       "import": "./dist/es/src/scalars/src\\scalars\\index.js",
       "types": "./dist/types/src/scalars/src\\scalars\\index.d.ts"
     },
-    "./EmailAddress": {
-      "require": "./dist/cjs/src/scalars/EmailAddress.js",
-      "import": "./dist/es/src/scalars/EmailAddress.js",
-      "types": "./dist/types/src/scalars/EmailAddress.d.ts"
+    "./src\\scalars\\EmailAddress": {
```
Comment (alejandrocabriales) [inline packages/scalars/package.json]: Could you clarify if this change was needed for compatibility or if it addresses a specific issue?

**Review by alejandrocabriales — CHANGES_REQUESTED** [review]

File: packages/scalars/package.json (hunk 1 above)
Comment (YasielCabrera) [reply packages/scalars/package.json]: It was an unwanted change. I rolled it back

Comment (alejandrocabriales) [reply packages/scalars/package.json]: LGTM

**Comment by alejandrocabriales** [comment]
LGTM

**Pull request merged**